### ISSUE TITLE
cyfmac55560-pcie.trxse version info was removed on latest tag

### ISSUE THREAD

**Issue by Noltari** [Infineon/ifx-linux-firmware#2]
**cyfmac55560-pcie.trxse** version information was removed on latest tag ([v5.10.9-2022_0909](https://github.com/Infineon/ifx-linux-firmware/blob/release-v5.10.9-2022_0909/firmware/versions)).

I guess that the previous version from v5.10.9-2022_511 still applies: https://github.com/Infineon/ifx-linux-firmware/compare/release-v5.10.9-2022_511...release-v5.10.9-2022_0909#diff-d437d25ebd3df8391fa4e186146f6354a979d02163ef5bbb9a8618ed53499345L73

**Comment by 0xraks** [comment]
Yeah, it's the same version.You can open the cyfmac55560-pcie.trxse in a text editor and find the info if you scroll all the way to the bottom.

But I think the file should not have been included since there is already a "cyfmax55572-pcie.trxse" file.

You can find the community release package here   which does not have that file: https://community.infineon.com/t5/Wi-Fi-Bluetooth-for-Linux/Cypress-Linux-WiFi-Driver-Release-FMAC-2022-09-09/td-p/370346
